.prettierrc.json

```diff
@@ -1 +1,14 @@
-{}
+{
+  "semi": true,
```

Comment:
Curious why you enabled these lint rules. Was this a file that wasn't meant to be merged? Do you feel these would be the best for our linting structure? Or is this personal preference?